path-manager/app/services/PathStore.scala

```diff
@@ -133,6 +199,18 @@ object PathStore extends Logging {
     }
   }
 
+  def deleteAliasesAssociatedWithPath(path: String) = {
```

Comment:
Now you've explained elsewhere this is Option rather than list/seq, I don't think this needs to be original suggestion...
```suggestion
  def deleteAlias(path: String) = {
```

Reply:
that said, based on your https://github.com/guardian/path-manager/pull/33#discussion_r468021596 if we want deletions only to happen direct in dynamo (I guess by one of us), then maybe we remove this function entirely for now and revisit if/when we need a feature for deleting aliases (where we implement the 'safe delete' I mentioned IRL yesterday - by checking CAPI Live or something)

Reply:
yeah, deleted.